lib/trino-plugin-toolkit/pom.xml

```diff
@@ -15,6 +15,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.test.parallel>instances</air.test.parallel>
```

Comment:
Any specific reason for this change ?

Reply:
Why do we need to run all the methods in same thread ?

Reply:
not needed know with refactored http server. Previously it was because overriding response by different test methods. reverted